README.md

```diff
@@ -1,217 +1,48 @@
-   ````
-   For Linux:   WILDFLY_HOME/bin/standalone.sh
-   For Windows: WILDFLY_HOME\bin\standalone.bat
-   ````
-3. To install the <span>Keycloak</span> adapter run the following commands:
-   ````
-   For Linux:
-
-     WILDFLY_HOME/bin/jboss-cli.sh -c --file=WILDFLY_HOME/bin/adapter-install.cli
-     WILDFLY_HOME/bin/jboss-cli.sh -c --command=:reload
-
-   For Windows:
-
-    WILDFLY_HOME\bin\jboss-cli.bat -c --file=WILDFLY_HOME\bin\adapter-install.cli
-    WILDFLY_HOME\bin\jboss-cli.bat -c --command=:reload
-   ````
-4. If you plan to try the SAML examples you also need to install <span>Keycloak</span> SAML adapter:
-
-   ````
-   For Linux:
+The quickstarts demonstrate securing applications with Keycloak. They provide small, specific, working examples
+that can be used as a reference for your own project.
 
-     WILDFLY_HOME/bin/jboss-cli.sh -c --file=WILDFLY_HOME/bin/adapter-install-saml.cli
-     WILDFLY_HOME/bin/jboss-cli.sh -c --command=:reload
+## Help and Documentation
 
-   For Windows:
+* [Documentation](https://www.keycloak.org/documentation.html)
+* [User Mailing List](https://lists.jboss.org/mailman/listinfo/keycloak-user) - Mailing list for help and general questions about Keycloak
+* [JIRA](https://issues.jboss.org/projects/KEYCLOAK) - Issue tracker for bugs and feature requests
 
-     WILDFLY_HOME\bin\jboss-cli.bat -c --file=WILDFLY_HOME\bin\adapter-install-saml.cli
-     WILDFLY_HOME\bin\jboss-cli.bat -c --command=:reload
-   ````
+## Reporting Security Vulnerabilities
 
-Integration tests
------------------
+If you've found a security vulnerability, please look at the [instructions on how to properly report it](http://www.keycloak.org/security.html)
 
-By default, the integration tests for each quickstart, expect this initial admin user to have `admin` as username and `admin` as password. This is configurable in each `ArquillianTest` class.
+## Reporting an issue
 
-```
-static {
-    try {
-        importTestRealm("admin", "admin", "/quickstart-realm.json");
-    } catch (IOException e) {
-        e.printStackTrace();
-    }
-}
-```
+If you believe you have discovered a defect in the quickstarts please open an issue in our [Issue Tracker](https://issues.jboss.org/projects/KEYCLOAK).
+Please remember to provide a good summary, description as well as steps to reproduce the issue.
 
-If you don't have access to admin's credentials, please import the `quickstart-realm.json` from `src/test/resources`.
+## Getting started
 
-To run integration tests for Wildfly managed quickstarts use the following command:
-```
-mvn clean install -Pwildfly-managed -Denforcer.skip=true
-```
+To get started refer to the [getting started](docs/getting_started.md) guide.
 
-If you want to run the tests for a quickstart that doesn't need Wildfly, you have to use appropriate profile. See a particular quickstart's README for more details.
+### Writing Tests
 
+To write tests refer to the [writing tests](docs/tests-development.md) guide.
```

Comment:
Broken link. Should be `docs/test-development.md` (without the S in "tests").

Reply:
Thanks, updated.